client/src/scenes/Battle.ts

```diff
@@ -317,21 +337,22 @@ export default class Battle extends Phaser.Scene {
 
   // create the enemies in the game, and design their behaviors
   private createEnemies() {
-    this.monsters = this.physics.add.group({
-      classType: Lizard,
-      createCallback: (go) => {
-        const lizardGo = go as Lizard;
-        lizardGo.body.onCollide = true;
-        lizardGo.setInteractive(); // Make the lizard interactive
-        lizardGo.on("pointerdown", () => {
-          if (!this.currentLizard) {
-            this.currentLizard = lizardGo;
-            this.showDialogBox(lizardGo);
-          } // Show dialog box when lizard is clicked
-        });
-      },
-    });
-    this.monsters.get(200, 123, "lizard");
+    this.monsters = []
+    // this.monsters = this.physics.add.group({
+    //   classType: Lizard,
+    //   createCallback: (go) => {
+    //     const lizardGo = go as Lizard;
+    //     lizardGo.body.onCollide = true;
+    //     lizardGo.setInteractive(); // Make the lizard interactive
+    //     lizardGo.on("pointerdown", () => {
+    //       if (!this.currentLizard) {
+    //         this.currentLizard = lizardGo;
+    //         this.showDialogBox(lizardGo);
+    //       } // Show dialog box when lizard is clicked
+    //     });
+    //   },
+    // });
```

Comment:
left this here just in case we decide to change monster to a group with interaction. Will delete soon if not used in later issue